.gitignore

```diff
@@ -4,6 +4,7 @@
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
 Azure.Functions.Cli
+azurite
```

Comment:
This is odd, it already exists in the feature branch

https://github.com/Azure/azure-functions-dotnet-worker/blob/80a2659320b7b20cf9697a2b6a9cda75e780185c/.gitignore#L7

Reply:
AH, you need to rebase. I updated feature/sdk-binding to be caught up with main